src/studying.md

```diff
@@ -219,8 +219,6 @@ mainly left around for compatibility with older Anki versions.
 
 - **Delete Note**: Deletes the note and all of its cards.
 
-- **Options**: Edit the options for the current deck.
```

Comment:
Why remove this?

Reply:
Because it appears twice in the manual.

Reply:
Ah sorry, I did not see that in the diff context.

Reply:
No problem :)